File: src/model/common/wallet_object.dart
class WalletObject {
  final int id;
  bool isLost = true;

  WalletObject(this.id);

  bool getIsLost() {
    return isLost;
  }

  void setIsLost(bool lost) {
    isLost = lost;
  }

  int getId() {
    return id;
  }
}

File: src/model/common/WalletObject.java
package model.common;

public class WalletObject {
    private final int id;
    private boolean isLost;

    public WalletObject(int id) {
        this.id = id;
    }

    public boolean isLost() {
        return isLost;
    }

    public void setLost(boolean lost) {
        isLost = lost;
    }

    public int getId() {
        return id;
    }
}
